### PR TITLE
004 - Create sign up page

### PR DESCRIPTION
**Pull request by Sindex42**

### CODE REVIEW

File: app/assets/javascripts/users.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (zopa-simon-cass) [inline app/assets/javascripts/users.js]: Delete this?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,14 +1,22 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Acebook</title>
+    <title>Acebook Team Rocket</title>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
```
Comment (scass91) [inline app/views/layouts/application.html.erb]: Do we need to include JS here?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,22 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Acebook</title>
+    <title>Acebook Team Rocket</title>
```
Comment (zopa-simon-cass) [inline app/views/layouts/application.html.erb:4]: Do we need to include JS here?

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    else
+      message = "Couldn't create account. " \
+                "Please make sure you are using a valid email" \
+                " and password and try again."
+
+      flash.now.alert = message
+      render :new
+    end
+  end
+
```
Comment (zopa-simon-cass) [inline app/controllers/users_controller.rb:22]: I think we need a test here for equality between password & password confirmation - thoughts?

**Review by zopa-simon-cass — CHANGES_REQUESTED** [review]

**Review by zopa-simon-cass — APPROVED** [review]

**Review by GabMat97 — APPROVED** [review]

**Pull request merged**